src/sentry/api/endpoints/group_similar_issues_embeddings.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from typing import Any, Mapping
+
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.group import GroupEndpoint
+from sentry.eventstore.models import GroupEvent
+from sentry.seer.utils import SimilarIssuesEmbeddingsRequest, get_similar_issues_embeddings
+from sentry.web.helpers import render_to_string
+
+logger = logging.getLogger(__name__)
+
+
+def get_stacktrace_string(exception: Mapping[Any, Any], event: GroupEvent) -> str:
+    """Get the stacktrace string from an exception dictionary."""
+    if not exception["values"]:
```

Comment:
It might be helpful to get all the frames using https://github.com/getsentry/sentry/blob/a502309b17446ad85a65d9b8f9b8cd9156293ebd/src/sentry/utils/event_frames.py#L184-L186

This function handles frames being in different parts of the error, and is more general.

Reply:
thank you! right now, I'm modeling the exception based on how it looks in this [table](https://docs.google.com/spreadsheets/d/1menVeYMQCxgE4mha8if4XRuM2PXfQxJsoo9F_7-eQTU/edit#gid=55932162), but I'll keep this in mind as the algorithm goes through iterations

Reply:
yeah, we should definitely change this logic - she's basically going off of my janky logic for the moment